README.md

```diff
@@ -65,17 +65,15 @@ Also see the `examples` folder.
 
 ## Release a new version
 
-Update the version in:
-
-* `src/beku/version.py`
-* `README.md` : version and pip install command.
-
-Update the CHANGELOG.
-Commit and tag.
-Build and publish:
-
-```sh
-rm -rf dist/
-python -m build --sdist --wheel .
-twine upload dist/*
-```
+A new release involves bumping the package version and publishing it to PyPI.
+The easiest way to publish to PyPI is to allow the release GitHub action to do it for you.
+This action is also the preferred way, as it will publish a verified package using PyPI attestations.
+
+To release a new version follow the steps below:
+
+1. Create a new Git branch. For example `release-1.2.3`.
+2. Update the version string (`1.2.3`) in: `src/beku/version.py` and`README.md`.
+3. Commit, push and create a PR.
+4. After the PR is merged, switch to the `main` branch and update it.
+5. On the `main` branch, create and push the release tag. For example: `git tag 1.2.3 -m 1.2.3 && git push --tags`.
```

Comment:
note: `--tags` _can_ be dangerous. We should either specifically push the newly created tag or use `--follow-tags` instead.

Reply:
[done](https://github.com/stackabletech/beku.py/pull/34/commits/04da7ed6672fb14edbbac60ff76eae2cda02e807)